lib/api/objectPutPart.js

```diff
@@ -113,6 +113,7 @@ function objectPutPart(authInfo, request, streamingV4Params, log,
                 log.debug('access denied for user on bucket', { requestType });
                 return next(errors.AccessDenied, destinationBucket);
             }
+            writeContinue(request, request._response);
```

Comment:
To avoid EPIPE error, should we `writeContinue` right before the data from the response is consumed.
What if, for example, the cipher bundle fails right after we indicate to our client that the request body should be sent? Won't it also lead to a client EPIPE error?

Reply:
it's true, this can happen, still will be trigger in response to an internal error so less problematic, but you are right, i will move the call

Reply:
modified :)